packages/harmony/src/components/input/TextInput/TextInput.stories.tsx

```diff
@@ -21,7 +21,7 @@ const StoryRender = (props: TextInputProps) => {
 }
 
 const meta: Meta<typeof TextInput> = {
-  title: 'Components/Input/TextInput',
+  title: 'Inputs/TextInput',
```

Comment:
I feel like the category titles should be singular, but i'm not sure why?

Reply:
Down to do that depending on what ux wants, just copied this from a storybook that did similar